response_operations_ui/views/surveys.py

```diff
@@ -34,7 +34,26 @@ def view_survey(short_name):
             "title": f"{survey_details['survey']['surveyRef']} {survey_details['survey']['shortName']}",
         }
     ]
+
+    # Mapping backend states to frontend sates for the user
+    for collection_exercise_state in survey_details["collection_exercises"]:
```

Comment:
we need to make this status mapping common as the same mapping is used in the collection exercise details page